recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conan.tools.cmake import CMake, CMakeToolchain
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import re
+
+required_conan_version = ">=1.37.0"
+
+class ConanFile(ConanFile):
+    name = "tbb"
```

Comment:
I would create a new recipe named `onetbb` and add the `deprecating` attribute to the current `tbb` recipe (this requires two different pull requests)

Reply:
I'm not quite sure about deprecating tbb, it might be a bit early to do that. I can certainly make this a separate package.